charts/hocs-toolbox/templates/_envs.tpl

```diff
@@ -10,12 +10,12 @@
   valueFrom:
     secretKeyRef:
       name: {{ $.Release.Namespace }}-trusted-s3
-      key: access_keys_ro_user
+      key: access_key_id
```

Comment:
If this key is used by hocs/toolbox then should it not be: `AWS_ACCESS_KEY_ID`

Reply:
This is the key that holds the AWS_ACCESS_KEY_ID in the k8s secrets
<img width="215" alt="Screenshot 2024-06-13 at 15 13 12" src="https://github.com/UKHomeOffice/hocs-helm-charts/assets/87995501/a1ca3842-3b2e-4ada-93a6-6c9b9bd5a585">

This is used to set AWS_ACCESS_KEY_ID in the script that access s3:
https://github.com/UKHomeOffice/hocs-toolbox/blob/main/scripts/download-migration-docs#L26-L29